codegen/src/botocore.rs

```diff
@@ -0,0 +1,4 @@
+#[cfg(feature = "serde_macros")]
+include!("botocore.rs");
```

Comment:
That's a bug! It should be botocore.in.rs. Because of our ongoing issues with Serde on nightly, I can't actually compile it on nightly and hence didn't notice this.